src/main/java/org/isf/medicalstockward/gui/WardPharmacyRectify.java

```diff
@@ -669,12 +672,15 @@ private JSpinner getJSpinnerNewQty() {
 	/**
 	 * @return
 	 */
-	private JLabel getJLabelStockQty() {
+	private JTextField getJLabelStockQty() {
 		if (jLabelStockQty == null) {
-			jLabelStockQty = new JLabel(""); //$NON-NLS-1$
+			jLabelStockQty = new JTextField(""); //$NON-NLS-1$
 			jLabelStockQty.setHorizontalAlignment(SwingConstants.CENTER);
 			jLabelStockQty.setPreferredSize(new Dimension(100, 25));
-			jLabelStockQty.setFont(new Font("Tahoma", Font.BOLD, 14)); //$NON-NLS-1$
+			jLabelStockQty.setFont(FONT_BOLD);
+			jLabelStockQty.setForeground(Color.BLACK);
+			jLabelStockQty.setEditable(false);
+			jLabelStockQty.setBorder(BorderFactory.createEmptyBorder());
 		}
 		return jLabelStockQty;
 	}
```

Comment:
should the variable also be renamed to jTextFieldStockQty? 

Anyway I was suggesting to not "parse" its text, rather get it directly from the source variable

Reply:
OK; made the name change.   Sorry I didn't understand your original comment; but given that it is a text field this just feels better.